draft-ietf-dnsop-svcb-https.md

```diff
@@ -1123,7 +1126,8 @@ single-client granularity.
 
 ## Interaction with Alt-Svc
 
-Clients that implement support for both Alt-Svc and HTTPS records SHOULD
+Clients that implement support for both Alt-Svc and HTTPS records and
+receive an Alt-Svc header field in a response SHOULD
```

Comment:
```suggestion
Clients that implement support for both Alt-Svc and HTTPS records and
are making a connection based on a cached Alt-Svc response SHOULD
```

Alt-Svc can come from an H2 frame not just a header.
The lookup also should come at request time, not at the time of receiving the Alt-Svc,
as these could be separated by hours or days.